### PR TITLE
[FSDE-12536] Migrate Progress component to new Vue documentation

### PR DESCRIPTION
**Pull request by josecarlosgonzalezv**

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-282/1/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://ba96b0a06181d7ffb8862b6c43f4e41811b1e589-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-282-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-282/2/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-282/3/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://25c364299db6166ea4fc19c397e3207e8a407edc-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-282-ux-chi.kubeodc-test.corp.intranet/ (internal)

File: src/documentation/pages/components/progress/index.vue [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<template lang="pug">
+  <div>
```
Comment (evansito) [inline src/documentation/pages/components/progress/index.vue:2]: could be replaced with pug's div

**Review by evansito — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-282/4/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://b845e5652cefd67ca716901ccb7adbd9919d4a6b-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-282-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Review by jllr — APPROVED** [review]

**Pull request merged**